bin/build.sh

```diff
@@ -19,13 +19,13 @@ PYTHON_VERSION=py3
 
 CKAN_GIT_VERSION=$CKAN_VERSION
 CKAN_GIT_ORG=qld-gov-au
+SOLR_VERSION=9
 
 if [ "$CKAN_VERSION" = "2.11" ]; then
-    CKAN_GIT_VERSION=ckan-2.11.1
+    CKAN_GIT_VERSION=ckan-2.11.2-qgov.1
```

Comment:
can we pull this to the matrix?

Reply:
> can we pull this to the matrix?

Not a great option. The matrix is designed to try every permutation.

Having values tied specifically to one matrix value means applying workarounds. It's easier to do it here.

Moreover, when running tests locally, it's an advantage to have `build.sh` manage everything instead of needing to specify everything as shell variables.